containerscan/datastructures.go

```diff
@@ -16,6 +16,13 @@ const (
 	CriticalSeverity   = "Critical"
 
 	ContainerScanRedisPrefix = "_containerscan"
+
+	UnknownScore    = 1
```

Comment:
No sure this is the right place since severity is not just for containers 
@Bezbran WDYT

Reply:
I would consider maybe to put these under identifiers or something more general

Reply:
do you think we need new file smth like `severities.go` just for those constants?

Reply:
The package is the main thing not the file.
I was thinking maybe it should be under identifiers next to the other contents - but I see that the map is also here - so let it be